pkg/daemon/ceph/client/pool.go

```diff
@@ -357,6 +357,12 @@ func CreateECPoolForApp(context *clusterd.Context, clusterInfo *ClusterInfo, poo
 		return errors.Wrapf(err, "failed to create EC pool %s. %s", poolName, string(output))
 	}
 
+	args = []string{"pool", "init", poolName}
+	output, err = NewRBDCommand(context, clusterInfo, args).Run()
+	if err != nil {
+		return errors.Wrapf(err, "failed to initialize EC pool %q. %s", poolName, string(output))
+	}
```

Comment:
`rbd pool init` would always fail here because you are calling it before (potentially?) enabling EC overwrites.

Reply:
Same comment as for the replicated case -- this needs to be done only for rbd pools and it would be nice to assert that `enableECOverwrite` is true in that case.  Otherwise `rbd pool init` would fail with "pool missing required overwrite support" error.